### ISSUE TITLE
Loss quickly becomes NaN

### ISSUE THREAD

**Issue by bob80333** [Xiaoming-Yu/DMIT#5]
When training using a slightly modified season_transfer code (see my fork), everything quickly becomes NaN on my dataset:

```
(epoch: 1, iters: 20, time: 1.415) , D_rand_prior: 3.942, G_rand_prior: 0.426, D_rand_enc: 3.968, G_rand_enc: 0.356, D_content: 1.079, G_content: 0.930, errRec: 2.804, errKl: 0.005, errStyle: 0.600, errContent: 1.191, errCyc: 3.077, errDiv: 0.298
(epoch: 1, iters: 40, time: 1.413) , D_rand_prior: 3.905, G_rand_prior: 0.371, D_rand_enc: 3.860, G_rand_enc: 0.659, D_content: 1.675, G_content: 2.137, errRec: 4.170, errKl: 0.003, errStyle: 0.836, errContent: 0.912, errCyc: 4.638, errDiv: 0.621
(epoch: 1, iters: 60, time: 1.409) , D_rand_prior: 3.911, G_rand_prior: 0.255, D_rand_enc: 3.873, G_rand_enc: 0.415, D_content: 0.725, G_content: 0.183, errRec: 3.414, errKl: 0.033, errStyle: 0.749, errContent: 0.760, errCyc: 3.771, errDiv: 0.406
(epoch: 1, iters: 80, time: 1.414) , D_rand_prior: 3.803, G_rand_prior: 0.625, D_rand_enc: 3.784, G_rand_enc: 0.628, D_content: 0.579, G_content: 0.180, errRec: 5.343, errKl: 0.018, errStyle: 0.949, errContent: 0.653, errCyc: 5.572, errDiv: 0.617
(epoch: 1, iters: 100, time: 1.416) , D_rand_prior: 3.807, G_rand_prior: 0.434, D_rand_enc: 3.769, G_rand_enc: 0.575, D_content: 0.616, G_content: 0.819, errRec: 3.402, errKl: 0.030, errStyle: 0.615, errContent: 0.664, errCyc: 4.447, errDiv: 0.444
(epoch: 1, iters: 120, time: 1.409) , D_rand_prior: nan, G_rand_prior: nan, D_rand_enc: nan, G_rand_enc: nan, D_content: nan, G_content: nan, errRec: nan, errKl: nan, errStyle: nan, errContent: nan, errCyc: nan, errDiv: nan
(epoch: 1, iters: 140, time: 1.408) , D_rand_prior: nan, G_rand_prior: nan, D_rand_enc: nan, G_rand_enc: nan, D_content: nan, G_content: nan, errRec: nan, errKl: nan, errStyle: nan, errContent: nan, errCyc: nan, errDiv: nan
(epoch: 1, iters: 160, time: 1.400) , D_rand_prior: nan, G_rand_prior: nan, D_rand_enc: nan, G_rand_enc: nan, D_content: nan, G_content: nan, errRec: nan, errKl: nan, errStyle: nan, errContent: nan, errCyc: nan, errDiv: nan
(epoch: 1, iters: 180, time: 1.409) , D_rand_prior: nan, G_rand_prior: nan, D_rand_enc: nan, G_rand_enc: nan, D_content: nan, G_content: nan, errRec: nan, errKl: nan, errStyle: nan, errContent: nan, errCyc: nan, errDiv: nan
```

Are there any hyperparams you recommend tweaking to fix this?  (My dataset has one domain of vector drawings, and one of images, if that's important)

**Comment by Xiaoming-Yu** [comment]
Can you provide some examples of traning data?

**Comment by bob80333** [comment]
I figured it out.  One of the domains was extracted from video, so there were many frames with nothing in them.   Once I removed those images from the dataset the loss stopped becoming NaN.  A similar dataset to what I have is [selfie2anime](https://drive.google.com/file/d/1xOWj1UVgp6NKMT3HbPhBbtq2A4EDkghF/view) from the U-GAT-IT paper.

Thanks for the quick response, it made me take another look at my dataset leading me to discover the problem.
